.travis.yml

```diff
@@ -4,6 +4,7 @@ sudo: false
 
 scala:
   - "2.12.4"
+  - "2.11.12"
```

Comment:
This way the mima checks are done in parallel

Reply:
Though now I need the sbt task to pick that scala version... I don't want to do + since that'd duplicate work